guides/using-component-harnesses.md

```diff
@@ -235,7 +235,7 @@ self-explanatory.
 
 The harnesses also make clear which option should be selected. Without the harness, you need a
 comment explaining what `options[1]` means, but with the `MatSelectHarness` you can use the `text`
-filter rather than the index and the code becomes self-documenting.
+filter rather than the index, and the code becomes self-documenting.
```

Comment:
This comma is also misplaced; it was already a bit of a run-on sentence, so I would either leave it as is or rewrite like
```markdown
Without the harness, you need a comment that explains what
`options[1]` means. With `MatSelectHarness`, however, the filter
API makes the code self-documenting.
```